### PR TITLE
BY-COVID track update

### PR DESCRIPTION
**Pull request by wm75**
@teresa-m this provides more context and fixes the issue with linking to the video **and** the infectious diseases toolkit showcase page for the "automated SARS-CoV-2 genome surveillance system built around Galaxy".

### CODE REVIEW

File: events/tracks/gta2024-bycovid.md [hunk 1]
```diff
@@ -37,12 +40,12 @@ program:
...
     description: |
-      An automated SARS-CoV-2 genome surveillance system built around Galaxy. If you encounter any issue please ask us on Slack. 
+      Learn about how to scale up your Galaxy-based viral sequencing data analysis to batches of samples and how to automate the processing of such batches for highest sample throughput. Watch the introductory video for an overview of the contents of this section and how the pieces are connected. 
     tutorials:
       - type: custom
-        name: "[An automated SARS-CoV-2 genome surveillance system built around Galaxy](https://www.infectious-diseases-toolkit.org/showcase/covid19-galaxy)"
+        name: "An automated SARS-CoV-2 genome surveillance system built around Galaxy"
         description: |
-          Lecture Video
+          [Lecture Video](https://gallantries.github.io/video-library/videos/one-health/galaxy-pathogen-surveillance); [Showcase page mentioned in the video](https://www.infectious-diseases-toolkit.org/showcase/covid19-galaxy)
```
Comment (hexylena) [inline events/tracks/gta2024-bycovid.md:48]: thanks for fixing this!

**Comment by teresa-m** [comment]
@wm75: thanks for working on the page. I added some changes for your tracks already here: https://github.com/galaxyproject/training-material/pull/5384
Guess we just need to see which PR to accept first so as to not overwrite your changes!

**Comment by wm75** [comment]
@teresa-m looks like you can simply merge this one here and remove all changes to the same file from #5384

**Review by shiltemann — APPROVED** [review]

**Comment by teresa-m** [comment]
@shiltemann: thanks a lot for merging it

**Comment by shiltemann** [comment]
@teresa-m no problem, just double check that everything looks ok once the changes are redeployed :)

**Comment by teresa-m** [comment]
@shiltemann it looks to me that all changes did make it. Thanks!

**Pull request merged**